src/main/java/uk/gov/hmcts/reform/sendletter/controllers/ReportController.java

```diff
@@ -34,6 +38,10 @@ public ReportController(
 
     @PostMapping(path = "/print-status-check")
     @ApiOperation(value = "Execute action for print state")
+    @ApiResponses({
+        @ApiResponse(code = 401, message = "Invalid service authorisation")
```

Comment:
401 is missing auth header. invalid is 403

Reply:
There's also validation whether service has access to perform action and response is the same - 401 😕 . Made assumption it's "invalid service auth" for both checks

Reply:
sooo.. `401 Unauthorised` and `403 Forbidden`. the for both assertions we use 401 and supply error message. initial post/get endpoints we have has `Invalid service authorisation header` as a message. but they don't check whether service is allowed to perform action so makes sense to include header.

tl;dr what message do you think it should be?